docs/exploring/Simplify-GenAI-RAG-Deployments-with-Open-Platform-for-Enterprise-OPEA.markdown

```diff
@@ -0,0 +1,44 @@
+
+OPEA treats each component in the RAG pipeline as a modular building block, ensuring easy interchangeability and customization. Say, for example, you’re using an LLM like [Mistral](https://huggingface.co/docs/transformers/en/model_doc/mistral), but want to easily replace it with [Falcon](https://huggingface.co/docs/transformers/en/model_doc/falcon). Or, say you want to replace a vector database on the fly. You don’t want to have to rebuild the entire application. That would be a nightmare. OPEA makes experimentation and deployment easy by providing robust tools and frameworks for seamless integration across platforms.
+
+## ChatQnA Reference Architecture
+
+Among the multiple [examples OPEA provides](https://github.com/opea-project/GenAIExamples) to help with deployments of GenAI applications ([CodeGen](https://github.com/opea-project/GenAIExamples/tree/main/CodeGen) or [VisualQnA](https://github.com/opea-project/GenAIExamples/tree/main/VisualQnA)), we will explore the RAG chatbot ([ChatQnA](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA)) as a prototypical example, composed of several independent modules, each tightly coordinated to handle specific functions or tasks, as shown in the image:
+
+![ChatQnA OPEA Blueprint]({{site.baseurl}}/assets/images/opea_arq.png)
+The reference architecture is divided into 3 parts:
+
+* **Conversational UI**: The chatbot interface includes an input field for user queries, which displays responses and passes the user inputs to the backend (Megaservice) or RAG context. 
+
+* **ChatQnA Megaservice**: This group of microservices receives the user message from the UI and starts the RAG process. The megaservice is a higher-level construct that assembles multiple microservices (retriever, embeddings, reranking, LLM) to deliver end-to-end applications. Unlike microservices, which handle specific tasks, this megaservice will orchestrate the microservices’ interactions to provide a complete solution, encapsulating complex business logic and workflows. This modular approach integrates microservices as a unit to fulfill specific application requirements.
+
+* **RAG Context**: The example allows the user to upload a file to be used as context for future queries. The dataprep microservice performs the preprocessing and inserts it into the vector database to be used to provide context for future queries.
+Each microservice is available as a Docker container. This allows any GenAI example to be deployed using Docker Compose or a Kubernetes cluster.
+
+##Call to Action
```

Comment:
Please put a space after the `##`. Otherwise, this is not interpreted as a section title.